test/converters/junitxml/junitxml_test.go

```diff
@@ -33,152 +33,265 @@ func Test_regroupErrors(t *testing.T) {
 		suites []junit.TestSuite
 		want   []junit.TestSuite
 	}{
-		{"regroup error message", []junit.TestSuite{
-			{TestCases: []junit.TestCase{
-				{Error: &junit.Error{Message: "error message"}},
-			}},
+		{
```

Comment:
pls add a case where message and value  has something

Reply:
or pls prove that it is invalid/forbidden

Reply:
also please add a test where a golden xml contains the field that our implementation currently doesn't support.

NOTE: please next time do the reproduction test case first - to prove the issue is existing and you have a test case that need to be fixed client side.

Reply:
There is a test case "should append system error, error message, error body to failure" that has both Failure content and message attribute

Reply:
Added test with XML.